recipes-support/libtinyxml/libtinyxml_2.6.2.bbappend

```diff
@@ -0,0 +1,6 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
```

Comment:
who provides libtinyxml?

Reply:
Its available in meta-openembedded/meta-oe/recipes-support

Reply:
In that case should we upstream the package config changes?

Reply:
The version currently included by meta-oe does not have this package config. But the higher version of the libtinyxml package have the package config support already (even with 2.6.2_2 itself have the support)

Reply:
I understand that, what I'm asking is should we send a patch to meta-openembedded morty to update the community with a similar change versus keeping it in meta-wpe?

Reply:
But currently RDK is using the its own fork of meta-openembedded from :https://code.rdkcentral.com/r/rdk/components/opensource/oe/meta-openembedded with rdk/morty branch. So in that case, pushing the changes to meta-openembedded:morty will not work. So do we need to push this to rdk/morty branch of rdk openembedded fork ?